numpy/random/_pcg64.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, Literal, Union
```

Comment:
`Literal` needs a conditional import, as it was only added to `typing` in python 3.8.

``` python
import sys
if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```